tests/system/system_test.go

```diff
@@ -22,15 +22,22 @@ func NewSystemTest(t *testing.T, dir string, vars map[string]string) *SystemTest
 	vars["endpoint"] = os.Getenv(config.ENVVAR_ENDPOINT)
 	vars["token"] = os.Getenv(config.ENVVAR_TOKEN)
 
+	t.Logf("[DEBUG] endpoint: %s", vars["endpoint"])
```

Comment:
We don't need the `[DEBUG]` prefix when logging using `*testing.T` since that only prints based on whether `go test -v` was passed. However, we do need them when logging using the `log` package, because that's how  we determine whether to [print statements or not](https://github.com/quintilesims/layer0/blob/api-refactor/common/logging/logging.go#L17).

[nit] Personally, I don't like having more than one way to log, but I feel it was/is better to use `log.Print` over `t.Log` in system tests (and only `log.Print` to keep things simple) given those logs print in realtime. The latter option forces you to wait until the test completes or errors before getting any feedback, which can be problematic given the length of time system tests can run.